### PR TITLE
Runpp 3ph linting cleanup

### PR DESCRIPTION
**Pull request by elodin**
In the 3ph files, the IDE is lighting up many lines because of minor code style warnings. This takes the attention away from real problems and makes it unnecessarily hard to work on these files. 

So this PR aims to remove all the linting issues without changing the functionality.

### CODE REVIEW

File: pandapower/pd2ppc_zero.py [hunk 1]
```diff
@@ -8,17 +8,18 @@
 from itertools import product
 
 import pandapower.auxiliary as aux
-import pandas as pd
+# import pandas as pd
```
Comment (rbolgaryn) [inline pandapower/pd2ppc_zero.py:11]: why is this line commented-out and not removed?

File: pandapower/pd2ppc_zero.py [hunk 2]
```diff
@@ -8,17 +8,18 @@
...
 from pandapower.build_bus import _build_bus_ppc
 from pandapower.build_gen import _build_gen_ppc
-#from pandapower.pd2ppc import _ppc2ppci, _init_ppc
+# from pandapower.pd2ppc import _ppc2ppci, _init_ppc
```
Comment (rbolgaryn) [inline pandapower/pd2ppc_zero.py:14]: and this one?

File: pandapower/pd2ppc_zero.py [hunk 3]
```diff
@@ -52,9 +53,10 @@ def _pd2ppc_zero(net, k_st, sequence=0):
     # generates "internal" ppci format (for powerflow calc) from "external" ppc format and updates the bus lookup
     # Note: Also reorders buses and gens in ppc
     ppci = _ppc2ppci(ppc, net)
-    #net._ppc0 = ppc    <--Obsolete. now covered in _init_ppc
+    # net._ppc0 = ppc    <--Obsolete. now covered in _init_ppc
```
Comment (rbolgaryn) [inline pandapower/pd2ppc_zero.py:56]: can be removed

File: pandapower/pf/runpp_3ph.py [hunk 4]
```diff
@@ -1,19 +1,20 @@
 # -*- coding: utf-8 -*-
 """
-# Copyright (c) 2016-2019 by University of Kassel and Fraunhofer Institute for Energy Economics
+# Copyright (c) 2016-2022 by University of Kassel and Fraunhofer Institute for Energy Economics
 # and Energy System Technology (IEE), Kassel. All rights reserved.
 
 @author: Shankho Ghosh (sghosh) (started Feb 2018)
```
Comment (rbolgaryn) [inline pandapower/pf/runpp_3ph.py:6]: we can remove the @author lines from here

**Review by rbolgaryn — CHANGES_REQUESTED** [review]

File: pandapower/pd2ppc_zero.py (hunk 1 above)
Comment (elodin) [reply pandapower/pd2ppc_zero.py:11]: The general approach of this PR is to only correct linting issues without removing code that might be helpful for future work on the files. This import could be deleted without a problem, but there should be another PR to clean unwanted/unneeded comments anyway.

File: pandapower/pd2ppc_zero.py (hunk 3 above)
Comment (elodin) [reply pandapower/pd2ppc_zero.py:56]: see above

File: pandapower/pd2ppc_zero.py (hunk 1 above)
Comment (rbolgaryn) [reply pandapower/pd2ppc_zero.py:14]: okay

**Review by rbolgaryn — APPROVED** [review]

**Comment by rbolgaryn** [comment]
@elodin can you please resolve the conflicts so that we can merge?

**Comment by elodin** [comment]
Solved the conflict @rbolgaryn and started merging. Another review is required though

**Review by rbolgaryn — APPROVED** [review]

**Pull request merged**